spring-boot/remote/src/main/java/org/infinispan/tutorial/simple/spring/remote/RemoteClientApp.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.tutorial.simple.spring.remote;
+
+import java.lang.invoke.MethodHandles;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * This example shows how to configure Spring Boot and Infinispan Server
+ */
+@SpringBootApplication
+public class RemoteClientApp {
+
+   private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+   @Autowired
+   private RemoteCacheManager cacheManager;
+
+   public static void main(String[] args) {
+      SpringApplication.run(RemoteClientApp.class, args);
+   }
+
+   @Bean
+   public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
+      return args -> {
+         RemoteCache<String, String> cache = cacheManager.getCache("default");
```

Comment:
Could the example also include how a second remote cache could also be `@AutoWired`? That way the example would show both `RemoteCacheManager` and `RemoteCache` auto wired. Thoughts?

Reply:
Why not! I will try that 💃